index.html

```diff
@@ -7,10 +7,48 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Kevin Salina Portfolio</title>
     <link rel="stylesheet" href="/styles/styles.css">
+    <script src="https://kit.fontawesome.com/421a2e0bfb.js" crossorigin="anonymous"></script>
```

Comment:
generally sometimes people like to load JS script towards the end of the file (in the body) but this script is impacting your UI and CSS, so I think its good to load it in the head tag.